packages/web-app/src/config/review-appeal-submission.js

```diff
@@ -6,6 +6,11 @@ const reviewOutcomeOption = {
   incomplete: '3',
 };
 
+const reviewOutComeById = Object.keys(reviewOutcomeOption).reduce((result, current) => {
+  const key = reviewOutcomeOption[current];
+  return Object.assign(result, { [key]: current });
+}, {});
```

Comment:
Not entirely sure we need to map the ids to a string value to use as a conditional in the code, we could just refer to the id as we've done in other places, it might be a bit more readable right now but if the db value changes then the meaning of this string value could change or be lost unless we go through the UI to change them. More of an observation rather than expecting it to be changed.